src/main/scala/viper/gobra/translator/util/VprInfo.scala

```diff
@@ -28,4 +29,19 @@ object VprInfo {
     val annotation = vpr.AnnotationInfo(Map(key -> values))
     vpr.ConsInfo(annotation, info)
   }
+
+  private def annotationToInfo(a: BackendAnnotation): vpr.AnnotationInfo = {
```

Comment:
"annotation" feels a bit overloaded because we also already have the Annotations for error backtranslation. How about using the more verbose "backendAnnotation", also for the other added methods in this object.